docs/operations/scheduler.md

```diff
@@ -0,0 +1,128 @@
+Additionally, you must  allow `storageclasses` to use the verbs: `watch`, `list` and `get`, you can add the following at the end of the 
+`clusterrole system:kube-scheduler`:
+
+```yaml
+- apiGroups:                                                                                                   
+  - storage.k8s.io                                                                                             
+  resources:                                                                                                   
+  - storageclasses                                                                                             
+  verbs:                                                                                                       
+  - watch                                                                                                      
+  - list                                                                                                       
+  - get 
+```
+
+
+## Scheduling VMs with the Custom Scheduler
+
+The second scheduler should be up and running. You can check it with:
+
+```shell
+$ kubectl get all -n kube-system
+```
+
+A deployment called `my-scheduler` have to be up and running if everything is properly configured.
+
+In order to launch VM/VMIs using the `my-scheduler`, you need to add `spec.SchedulerName: my-scheduler` to the VM
+definition. Here is an VM definition example:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  labels:
+    kubevirt.io/vm: vm-fedora-vm
+  name: vm-fedora
+spec:
+  running: true
+  template:
+    metadata:
+      labels:
+        kubevirt.io/domain: vm-fedora
+        kubevirt.io/vm: vm-fedora
+    spec:
+      schedulerName: my-scheduler
+      domain:
+        cpu:
+          cores: 1
+          sockets: 1
+          threads: 1
+        devices:
+          disks:
+            - name: containerdisk
+              disk:
+                bus: virtio
+            - name: cloudinitdisk
+              disk:
+                bus: virtio
+          interfaces:
+            - name: default
+              masquerade: {}
+          rng: {}
+        resources:
+          requests:
+            memory: 1Gi
+      networks:
+        - name: default
+          pod: {}
+      terminationGracePeriodSeconds: 180
+      volumes:
+        - containerDisk:
+            image: quay.io/containerdisks/fedora:latest
+          name: containerdisk
+        - cloudInitNoCloud:
+            userData: |-
+              #cloud-config
+              chpasswd:
+                expire: false
+              password: password
+              user: fedora
+          name: cloudinitdisk
+```
+In case the schedulerName does not match any existing scheduler, the `virt-launcher` pod will stay in a [pending](https://kubernetes.io/docs/tasks/extend-kubernetes/configure-multiple-schedulers/#verifying-that-the-pods-were-scheduled-using-the-desired-schedulers) state 
```

Comment:
I would put it like this:
> In case the specified `SchedulerName` does not match any existing scheduler, the `virt-launcher` pod will stay in in state `Pending`, until the specified scheduler can be found.